### ISSUE TITLE
ngAria - aria-disabled attribute not added on disabled textbox

### ISSUE THREAD

**Issue by nutangwari** [angular/angular.js#15155]
Hi Currently am using angular  v1.5.8 and i can see few of accessibility test cases are started failing after angular upgrade from 1.4.7
if am using a disabled textbox, aria attribute _aria-disabled="true"_ is not getting added to the textbox.
which was previously working fine. I couldn't find anything related to this in change logs.

**Comment by gkalpak** [comment]
This is because `ngAria` does not (unnecessarily) set attributes on native controls (which are accessible out-of-the-box).

This is documented in the [breaking changes section][1] for verison 1.5.0.
The [migration guide][2] has more details.

[1]: https://github.com/angular/angular.js/blob/master/CHANGELOG.md#user-content-150-ennoblement-facilitation-2016-02-05
[2]: https://docs.angularjs.org/guide/migration#migrating-from-1-4-to-1-5

**Comment by nutangwari** [comment]
Thanks for the info **Georgios Kalpakas**.
